my_offline_eval.py

```diff
@@ -163,6 +164,17 @@ def timeout_handler():
     pickle.dump(logs, open( f"logging/{game_id}.p", "wb" ) )    
     return game_won, game_id
 
+          
+def pair_two_agents_and_play_one_iterated_game(task) -> List[Union[bool, str]]:
+    levels = ["1", "2", "3", "4]
```

Comment:
typo, there is no closing "
We'd better test before merging....

Reply:
ah lol, didn't see the title

Reply:
Thanks for the catch!
It is my first time using that PR interface on a patch and so on, do you reckon I should "Resolve the conversation" to apply your changes, please?